sw/airborne/arch/chibios/modules/uavcan/uavcan.h

```diff
@@ -30,6 +30,17 @@
 #include <string.h>
 #include "mcu_periph/can.h"
 #include "modules/core/threads.h"
+#include "utils/circular_buffer.h"
+
+
+#ifndef UAVCAN_TX_FIFO_SIZE
+#define UAVCAN_TX_FIFO_SIZE 1024
+#endif
+
+#ifndef UAVCAN_MSG_MAX_SIZE
+#define UAVCAN_MSG_MAX_SIZE 256
```

Comment:
Isn't there a define in UAVCAN/DRONECAN itself that contains it from the specification?

Reply:
I don't remember any max size being specified, but maybe I'm wrong ?